WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -68,17 +72,13 @@ - (void)track:(WPAnalyticsStat)stat withProperties:(NSDictionary *)properties
 
 - (void)beginSession
 {
-#ifdef TRACKS_ENABLED
-    [self.tracksService switchToAnonymousUserWithAnonymousID:self.anonymousID];
-#endif
-    [self refreshMetadata];
-}
+    if (self.loggedInID.length > 0) {
```

Comment:
That was a holdover from when we were not including Tracks in the App Store builds, only the Internal Betas. That flag was removed from the build but apparently forgot to do it here too.